lib/jsdom/browser/Window.js

```diff
@@ -576,6 +577,8 @@ function Window(options) {
     return window._document.getSelection();
   };
 
+  this.customElements = CustomElementRegistry.create(window);
```

Comment:
This should be a getter; remember we don't have webidl2js generated code to handle that for us here.